server/real_estate/src/main/java/com/realestate/real_estate/repos/Appointment.java

```diff
@@ -2,8 +2,13 @@
 
 import lombok.Builder;
 import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
 import lombok.extern.jackson.Jacksonized;
 import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+@Getter
+@Setter
 @Data
```

Comment:
This @Data tag automatically adds getters and setters for you. Is there a reason why you need @Getter and @Setter?